### ISSUE TITLE
Elevated plasma dodecanoylcarnitine, C12:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5968]
**Preferred term label:** Elevated plasma dodecanoylcarnitine, C12:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Dodecanoylcarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
